### ISSUE TITLE
UIWebView issue

### ISSUE THREAD

**Issue by mkdevel** [TruckMovers/cordova-plugin-remote-injection#46]
Plugin is not working any more. Now if I add this plugin to iOS project, I get following error from app store: ITMS-90809: Deprecated API Usage - New apps that use UIWebView are no longer accepted. Instead, use WKWebView for improved security and reliability. 

Can you remove UIWebView components from plugin? 
Problems are all because of new App store requirements for any future update.

**Comment by markdon** [comment]
This plugin already supports WKWebView. Are you sure you've configured your Cordova project to use WKWebView? 

Maybe it's changed since I last looked, but a new Cordova project will run with UIWebView unless you install a plugin that changes the web view to WKWebView. (Unrelated to this plugin)

**Comment by mkdevel** [comment]
Yes, before it gave just warnings about that, but now it shows it as error from uploading to app store. Im using WKWebView (as shown in config.xml below) and if I remove only your plugin, then it uploads to app store. It seems that app store gives error even if there is any UIWebView code in plugin.

`<platform name="ios">
        <preference name="WKWebViewOnly" value="true" />
        <preference name="CordovaWebViewEngine" value="CDVWKWebViewEngine" />
        <feature name="CDVWKWebViewEngine">
            <param name="ios-package" value="CDVWKWebViewEngine" />
        </feature> 
</platform>
<plugin name="cordova-plugin-wkwebview-engine" />`

**Comment by markdon** [comment]
If you're sure it's this plugin, I would have a look at the more recent forks here: https://github.com/TruckMovers/cordova-plugin-remote-injection/network
to look for anyone what has already fixed the issue. If it's not fixed in a fork, you probably need to fix it yourself. There's a slight chance if you open an issue on one of the forks, they might fix the issue if they are still actively using the plugin.

**Comment by samall76** [comment]
> Yes, before it gave just warnings about that, but now it shows it as error from uploading to app store. Im using WKWebView (as shown in config.xml below) and if I remove only your plugin, then it uploads to app store. It seems that app store gives error even if there is any UIWebView code in plugin.
> 
> `<platform name="ios"> <preference name="WKWebViewOnly" value="true" /> <preference name="CordovaWebViewEngine" value="CDVWKWebViewEngine" /> <feature name="CDVWKWebViewEngine"> <param name="ios-package" value="CDVWKWebViewEngine" /> </feature> </platform> <plugin name="cordova-plugin-wkwebview-engine" />`

Did you use this: https://cordova.apache.org/announcements/2019/11/25/cordova-ios-release-5.1.0.html 
Same problem, but fixed it with 5.1.x.

**Comment by mkdevel** [comment]
Also after I cloned your projects and removed UI components, then app store approved my project and everything works fine.

**Comment by markdon** [comment]
@mkdevel it would be very good of you to fork the repository and commit the changes you've made to help others that find this issue.

Or just provide more details about the changes you made if they're simple.

**Comment by davidgovea** [comment]
I added some links to various forks in PR #47.

I find the [network view](https://github.com/vinq1911/cordova-plugin-remote-injection/network) very handy to see the most recent forks & activity.

**Comment by fnicollet** [comment]
As a side note on @davidgovea PR, the forks can be used in Cordova, by replacing the plugin name in package.json with the URL of a fork:
```
github:RobertY9/cordova-plugin-remote-injection
```
I was able to pass certification with this setting

Thanks @davidgovea !
Fabien
